oeps/architectural-decisions/oep-0058-arch-translations-management.rst

```diff
@@ -0,0 +1,210 @@
+  security issue we cannot control or debug.
+* Edx-transifex-bot is a black box: Login/scripts for edx-transifex-bot cannot be found
+  on GitHub or on wiki/secrets pages, and it is difficult to observe what work it is
+  supposed to do, or whether it is doing it correctly.
+* The translations for the Open edX Maple Release were never uploaded to Transifex,
+  because the automation handled by the edx-transifex-bot never uploaded it.
+* The translations for the Open edX Nutmeg Release were nearly not uploaded because the
+  edx-transifex-bot did not automatically upload it. In addition, the script for
+  uploading the translations did not work, and the script had to be tinkered for the new
+  Transifex API in order to upload it in time for release. The files uploaded were still
+  not in the preferred format because the new version of the Transifex API does not have
+  all the same features as the previous API version.
+
+  * Running the script to upload translations requires Transifex admin rights, and there
+    are few people with Admin rights to Transifex and knowledge of the Transifex API. We
+    should use the solution provided by Transifex to solve this type of problem.
+
+Rationale
+*********
+
+* This is an upgrade of a system we use regularly, but do not want to have to maintain
+  regularly.
+* Upgrading from a bot (machine user) to an app/workflow is recommended by GitHub and
+  makes the translation process more open source. 
+* The GitHub Transifex Integration App is developed and supported by Transifex
+* The Github Transifex App is very simple to configure and with many options. We can set
+  Transifex Projects to automatically upload/download translation files from a repository
+  once the translations are reviewed and accepted.
+* Transifex only allows a one-to-one relationship between repositories and Transifex
+  Projects. Organizing all of the translation files into one repository and one Transifex
+  Project has a lower cost: we pay by the project so we end up paying less, and by
+  putting all translation files in the same place, it is easier to track translation
+  progress, and debug translation issues.
+* A repository that only contains text/binary files, and uses branches to separate
+  translations related to Open edX releases can make all interactions with translations
+  very quick and simple due to the ability to clone the branch of a specific release,
+  with translations organized by repository name.
+* By using an app that is maintained by Transifex the organization, we reduce the
+  maintenance burden and are more future proof of changes they might make since they
+  maintain both the API and the github App.
+
+Proposed Implementation
+***********************
+
+Move translation files to a new repo
+====================================
+
+Translation files (of types .mo and .po) currently exist amongst the code/documentation
+they translate. We will move these Translation files from being amongst the
```

Comment:
I'm not sure I understand why "Translation files" is a proper noun... it reads oddly. My suggestion is to change all of these to "translation files".

Reply:
Translation Files has been demoted to translation files where appropriate.